Source/WebKit/UIProcess/Cocoa/WebProcessPoolCocoa.mm

```diff
@@ -240,6 +240,26 @@ static void registerUserDefaults()
     return cachedLockdownModeEnabledGlobally;
 }
 
+static std::optional<HashMap<String, URL>>& userInstalledFontURLs()
```

Comment:
I think we might want to add "shared" in the names to make it clear these are singleton.